modules/API/external/external_api.js

```diff
@@ -1,5 +1,8 @@
 import EventEmitter from 'events';
-import postisInit from 'postis';
+
+import PostMessageTransportBackend
+    from '../../transport/PostMessageTransportBackend';
+import Transport from '../../transport/Transport';
```

Comment:
I'm not comfortable with transport/index.js defining a `const transport = new Transport` and then this external_api.js doing it's own `new Transport`. I find it hard to understand. An index.js is used to turn a directory into a package (in the terms of Java, for example) so it's presence makes me feel like we'd be `import`ing the package elsewhere (like we do in react/) but then that's not happening here. It gives me a thing to think about that I shouldn't be thinking about.